services/121-service/src/connection/connection.service.ts

```diff
@@ -465,6 +469,36 @@ export class ConnectionService {
     return note;
   }
 
+  public async updateAttribute(
+    referenceId: string,
+    attribute: AttributesEnum,
+    value: string,
+  ): Promise<ConnectionEntity> {
+    const connection = await this.findOne(referenceId);
+    console.log('typeof connection[attribute]: ', typeof connection[attribute]);
+    let newConnection;
+
+    if (typeof connection[attribute] === 'undefined') {
+      const errors = 'This attribute is not known.';
+      throw new HttpException({ errors }, HttpStatus.NOT_FOUND);
+    }
+
+    if (typeof connection[attribute] === 'number' && Number.isInteger(value)) {
```

Comment:
A type of "`number`" would not always have to be an integer... I think..

But for now this will work...

Reply:
outdated now